client/src/static/mockedDataDemo.ts

```diff
@@ -40,7 +42,8 @@ export const buildInitialModelsList = ({ SIR_PN, SIR_FN, SEIR_PN, SEIRD_PN, SIRD
           },
         },
         {
-          file: '',
+          donuType: Donu.Type.EASEL,
```

Comment:
I suppose this is temporary because we will be only executing PNCs and GrFNs for the demo, not EASEL types.

Reply:
This is based ont the model name and type from Donu `sir.easel`.

Reply:
Yes, but we won't be really executing `easel` type models. We will be executing `gromet-pnc` and `gromet-fn`. Currently, only `gromet-pnc` are in donu. Does this make sense to you?